.pylintrc

```diff
@@ -0,0 +1,589 @@
+[MASTER]
```

Comment:
Please either add a minimalistic file (so that we can see what exactly you changed compared to to the default version) or commit the unchanged, vanilla version in a separate commit, then change what you need to be changed in a subsequent commit (again, so that we can see what exactly you changed compared to to the default version).

Reply:
Alright

Reply:
ok

Reply:
One last, tiny nitpick: Would be nice to know which version of pylint you have used to generated the file. Maybe adding this information to the commit message?